.styleci.yml

```diff
@@ -2,4 +2,10 @@ preset: psr2
 
 enabled:
   - long_array_syntax
-  - duplicate_semicolon
\ No newline at end of file
+  - duplicate_semicolon
+
+finder:
+  not-name:
+    # exclude PHP7 illegal annotation classes to avoid false alert. those must not be attempted to be used in PHP 7
+    - lib/Doctrine/ODM/PHPCR/Mapping/Annotations/Float.php
+    - lib/Doctrine/ODM/PHPCR/Mapping/Annotations/String.php
```

Comment:
@GrahamCampbell i did this for two of the 3 classes. but i still get the error for the classes that should be ignored. can you see what is wrong with this?